aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -1195,6 +1195,9 @@ static Tensor& logsumexp_out_impl(Tensor& result, const Tensor& self, IntArrayRe
 }
 
 Tensor& logsumexp_out(const Tensor& self, IntArrayRef dims, bool keepdim, Tensor& result) {
+  TORCH_CHECK(at::isFloatingType(result.scalar_type()),
+              "Expected floating point type for result tensor, got: ",
```

Comment:
start error messages with the name of the operator. `logsumexp(): Expected...`

Reply:
Addressed!